docs/man/skuba.1.md

```diff
@@ -17,7 +17,9 @@ reconfiguration in an easy way.
   Print usage statement.
 
 **-v**
-  set verbosity level.
+  set verbosity level [0-10] (default 0).
+  Most of Skuba INFO message appears in the level of 1
+  Regardless of INFO levels, ERROR and WARN messages will appear
```

Comment:
I might make this match the `-h` `number for the log level verbosity [0-10] (default 0)` to be consistent. I think the end goal is to have these man pages generated by the code anyhow and where most of the messages appears would be hard to track and could change easily without updating the man pages.

Reply:
How do you know the range is [0-10]?

Reply:
I searched all Skuba code.
These are a few examples for levels.
```
klog.V(0)
No
klog.V(1): Every Skuba
klog.V(1).Infof("=== applying state %s ===", stateName)
klog.V(1).Infof("=== state %s applied successfully ===", stateName)
klog.V(1).Infof("applying %q addon", addon.addon)
klog.V(2) : Vendor
klog.V(2).Infof("fetching Kubernetes version from URL: %s", url)


klog.V(3): vendor
klog.V(3).Infof("Couldn't find the LC_ALL, LC_MESSAGES or LANG environment variables, defaulting to en_US")
klog.V(3).Infof("Skipping service %s due to clusterIP = %q", svcName, service.Spec.ClusterIP)
klog.V(4): vendor
klog.V(4).Infof("Skipping: no address family match for %q on interface %q.", ip, intf.Name)
log.V(4).Infof("Got a Retry-After %ds response for attempt %d to %v", seconds, retries, url)
klog.V(5): vendor
klog.V(5).Infof("Unable to decode an event from the watch stream: %v", err)
klog.V(5).Infof("body was not decodable (unable to check for Status): %v", err)
klog.V(6): Vendor
klog.V(6).Infof("Copying %s", s)
klog.V(6).Infoln("Config loaded from file: ", filename)
klog.V(7): vendor
case bool(klog.V(7)):
   rt = newDebuggingRoundTripper(rt, debugJustURL, debugRequestHeaders, debugResponseStatus)
klog.V(8): vendor
case bool(klog.V(8)):
   max = 1024
}
klog.V(9): vendor
case bool(klog.V(9)):
   rt = newDebuggingRoundTripper(rt, debugCurlCommand, debugURLTiming, debugResponseHeaders)
klog.V(10):vendor
klog.V(10).Infof("ParseInt failed for value %+v in label %+v, %+v", ls.Get(r.key), ls, err)
klog.V(10).Infof("Invalid values count %+v of requirement %#v, for 'Gt', 'Lt' operators, exactly one value is required", len(r.strValues), r)
klog.V(10).Infof("returning cached discovery info from %v", filename)
```

Reply:
Just to add to the conversation, there is some documentation upstream regarding this:

- https://kubernetes.io/docs/reference/kubectl/cheatsheet/#kubectl-output-verbosity-and-debugging
- https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md

We have been trying to follow upstream approach here to be consistent.